### PR TITLE
Add Stubs For All Exercises

### LINKED ISSUE #145: Add stub solutions for all exercises
There should be a stub solution for every exercise, roughly like the following:

```erl
-module($module_name).

-export([fun_a/0, fun_b/2]).

fun_a() -> undefined.

fun_b(Param1, Param2) -> undefined.

test_version -> 1. % or whatever version is needed to pass the tests
```

Also we need a check in [`bin/journey-test`](https://github.com/exercism/xerlang/blob/master/bin/journey-test.sh) that fails if it is missing. 

```bash
if [[ ! -f "${exercism_exercises_dir}/erlang/${exercise}/src/${module}.erl ]]; then exit 1; fi
```

Should be a sufficient test, best put between current lines 201 (introduction of `$module`) and 202 (overwriting any stub that might exist with the example solution).

### PR DESCRIPTION
**Pull request by JordanAdams**
FIX #145 

This adds a stub module for all current exercises. The stub modules simply define a working `test_version/1` and stub definitions for functions required by the test suite.

For this change to work I've also modified the way `exercism.hrl` determines the correct module name. Instead of checking for the existence of the module file (which will now always be there). It instead checks for the existence of `example.erl` and uses that if it exists.

### CODE REVIEW

**Comment by NobbZ** [comment]
I start a review soon, it was a little bit too late yesterday, and I wasn't even able to think clearly after beeing at the dentist for an hour or so ;)

First thing: Can you please change "related issue" to "FIX"? That way the other issue will be closed automatically when we merge this.

File: exercises/accumulate/include/exercism.hrl [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
-    true  -> Module;
-    false -> example
+  case lists:member("example.erl", Files) of
+    true  -> example;
+    false -> Module
   end.
 
 version_test() ->
-    ?assertMatch(?TEST_VERSION, ?TESTED_MODULE:test_version()).
+  ?assertMatch(?TEST_VERSION, ?TESTED_MODULE:test_version()).
```
Comment (NobbZ) [inline exercises/accumulate/include/exercism.hrl]: Can we aggree on having a newline at the end of the files? It makes `cat` and paste much easier on a terminal ;)

**Comment by NobbZ** [comment]
Thank you!

File: exercises/accumulate/include/exercism.hrl (hunk 1 above)
Comment (JordanAdams) [inline exercises/accumulate/include/exercism.hrl]: @NobbZ All files should have trailing newlines now :)

**Pull request merged**